.github/scripts/check-yaml.py

```diff
@@ -253,12 +247,65 @@ def check(self) -> int:
                             message="The file must be non-empty",
                         )
 
+                if "document" in parsed:
+                    repo = parsed["document"].get("repo")
+                    commit = parsed["document"].get("commit")
+                    patterns = parsed["document"].get("patterns")
+
+                    line = self.get_yaml_line(
+                        content=content,
+                        taxonomy_path=taxonomy_path,
+                        yaml_path=".document")
+
+                    if repo.startswith("https://github.com/"):
+                        repo = repo.replace(
+                            "https://github.com/",
+                            "https://raw.githubusercontent.com/"
+                        )
+                        for file_path in patterns:
+                            if "*" in file_path:
+                                continue
+                            url = f"{repo}/{commit}/{file_path}"
```

Comment:
This will probably fail if the user supplies a proper git URL ending in `.git`.

Reply:
yup, I looked at our summit bounty submissions and thought we could start with something simple, but agree this is a narrow use case